features/step_definitions/journey/registration_steps.rb

```diff
@@ -0,0 +1,70 @@
+# Group pieces together when they are always consecutive (renewal - new lower tier - new upper tier)
+# Helpers vs Step to group things together - Choose helpers, avoid steps.
+# Helpers vs Step - repeat helpers, not steps.
+# Use `Given` to set up instance variables for the rest of the journey decisions
+
+Given("I want to register as a lower tier carrier") do
+  load_all_apps
+
+  @tier = :lower
+end
+
+When("I start a new registration journey in {string} as a {string}") do |location, organisation_type|
```

Comment:
Happy with using Cucumber format here for `{string}`. It is inconsistent with some other steps though, which use a regex: https://github.com/DEFRA/waste-carriers-acceptance-tests/blob/master/features/step_definitions/back_office/upper_tier/finance_admin_steps.rb#L44

Do you think we should use the regex format for consistency? I don't have a strong opinion either way on this. The regex is more powerful but harder to read!

Reply:
I think I do prefer the cucumber way TBH. Let's decide it later and fix the rest?

Reply:
Fine by me 👍